backend/main.go

```diff
@@ -58,6 +58,16 @@ func main() {
 		log.Warn().Msg("API_KEY is not set")
 	}
 
+	telegramChatID, ok := os.LookupEnv("TELEGRAM_CHAT_ID")
+	if !ok {
+		log.Warn().Msg("TELEGRAM_CHAT_ID is not set")
+	}
+
+	telegramUrl, ok := os.LookupEnv("TELEGRAM_URL is not set")
+	if !ok {
+		log.Warn().Msg("TELEGRAM_URL is not set")
+	}
```

Comment:
Since we want to support other kind of alerting system, we shouldn't make this as a `warn` log if the value does not exists.